WISE-Community/WISE
# Show ownership and license info on project details dialogs

To test:
1. Create a new unit using the authoring tool.
2. Go to `/teacher/library/personal`, find the new unit and click on it to open the project details dialog.
3. Verify that license info and CC BY-SA logo is displayed at the bottom and that `license.txt` file opens when clicking on 'View License' link.
4. Share unit with a new teacher and edit the unit as the shared teacher.
5. Go to `/teacher/library/personal` and click on the unit. Verify that both authors are displayed in the license info.
6. Copy the new project and click on the copied project to open project details dialog.
7. Verify that license info is displayed and that updated `license.txt` loads. Make sure the license info states that unit is a copy of the original unit and displays original authors.
8. Make a change to unit using the authoring tool. Go to `/teacher/library/personal` and click on the copied unit. Verify that license info shows the new author and states that unit is a derivative of the original unit. Verify that updated `license.txt` loads.
9. Repeat steps 6-8 but do so with a unit copy that's created by setting up a run with the original unit.
10. Click on an old unit that doesn't have a `license.txt` file. Verify that license info still shows up in the project details dialog but instead of 'View License' link, there is a 'More' link.

Resolves #1461.
